reference_designs/ipu-xrt/matrix_vector_multiplication/aie2.py

```diff
@@ -103,8 +103,9 @@ def device_body():
                     2,
                     memRef_A_ty,
                     [
-                        (k_in_i32s, 1),
-                        (m, k_in_i32s),
+                        (1, 1),
+                        (1, 1),
```

Comment:
(k, 1)

Reply:
i'm still figuring this one out... not landing yet